### PR TITLE
feat(pricing): better world name displaying for cn servers

### PR DESCRIPTION
**Pull request by zhyupe**
Add l12n to CN server names in marketboard and pricing mode (display Chinese words like '沃仙曦染' instead of pinyin 'WoXianXiRan'):

image
image

### CODE REVIEW

File: apps/client/src/app/pipes/pipes/world-name.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+@Pipe({
+  name: 'worldName'
+})
+export class WorldNamePipe implements PipeTransform {
+
+  constructor(private data: LocalizedDataService) {
+  }
+
+  transform(world: string): I18nName {
+    console.log('world', world)
```
Comment (Supamiu) [inline apps/client/src/app/pipes/pipes/world-name.pipe.ts:14]: Oops 😉

File: apps/client/src/app/modules/pricing/pricing/pricing.component.ts [hunk 2]
```diff
@@ -226,12 +226,12 @@ export class PricingComponent implements AfterViewInit {
         const amount = this.pricingService.getAmount(list.$key, row, false);
         let priceString: string;
         if (price.hq > 0 && amount.hq > 0) {
-          priceString = `${price.hq.toLocaleString()}gil x${amount.hq}(HQ) (${price.hqServer})`;
+          priceString = `${price.hq.toLocaleString()}gil x${amount.hq}(HQ) (${this.getWorldName(price.hqServer)})`;
```
Comment (Supamiu) [inline apps/client/src/app/modules/pricing/pricing/pricing.component.ts:229]: Why not use the pipe instead?

**Review by Supamiu — CHANGES_REQUESTED** [review]

File: apps/client/src/app/pipes/pipes/world-name.pipe.ts (hunk 1 above)
Comment (zhyupe) [reply apps/client/src/app/pipes/pipes/world-name.pipe.ts]: Oops, fixed

File: apps/client/src/app/modules/pricing/pricing/pricing.component.ts (hunk 2 above)
Comment (zhyupe) [reply apps/client/src/app/modules/pricing/pricing/pricing.component.ts:229]: Referencing to codes like `this.i18n.getName(this.l12n.getItem(row.id))`, I think it's reasonable (not perfect, sure). Also calling pipes will result in 2 more dependency injections.

**Review by Supamiu — APPROVED** [review]

**Pull request merged**